packages/create/src/recipes/remix/template/.eslintrc.cjs

```diff
@@ -43,10 +43,13 @@ module.exports = {
           { name: "Link", linkAttribute: "to" },
           { name: "NavLink", linkAttribute: "to" },
         ],
+        "import/resolver": {
+          typescript: {},
+        },
       },
     },
 
-    // TypeScript
+    // Typescript
```

Comment:
TypeScript is correct. (see https://www.typescriptlang.org/)

Reply:
ah ok, this is changed because newest Remix create will create this, so I copied their newest  .eslintrc.cjs

Reply:
😅 Fine to keep it then, not a big deal.